src/test/java/com/manywho/services/sql/ServiceFunctionalTest.java

```diff
@@ -94,6 +94,22 @@ protected void deleteTableIfExist(String tableName, Connection connection){
         }
     }
 
+    protected void deleteViewIfExist(String tableName, Connection connection){
+        try {
+            String sql;
+            switch (DbConfigurationTest.databaseTypeForTest) {
+                case "sqlserver":
+                    sql = "DROP VIEW "+ escapeTableName(tableName);
+                    break;
+                default:
+                    sql = "DROP VIEW IF EXISTS "+ escapeTableName(tableName);
+            }
+
+            connection.createQuery(sql).executeUpdate();
+        } catch (Exception ex){
```

Comment:
This exception should be logged, otherwise something could be breaking and we'd have no idea what's going on. Basically, you should never have an empty `catch` block, anywhere